service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/SkipBucketCloneStep.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.bucket;
+
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.common.utils.FlightUtils;
+import bio.terra.workspace.generated.model.ApiClonedControlledGcpGcsBucket;
+import bio.terra.workspace.generated.model.ApiCloningInstructionsEnum;
+import bio.terra.workspace.service.resource.controlled.cloud.gcp.gcsbucket.ControlledGcsBucketResource;
+import org.springframework.http.HttpStatus;
+
+/**
+ * If cloning instructions say COPY_NOTHING, we need to stub the response object and add
+ * it to the flight context. This step assumes we are indeed copying nothing.
+ */
+public class SkipBucketCloneStep implements Step {
+
+  private final ControlledGcsBucketResource sourceBucket;
+
+  public SkipBucketCloneStep(ControlledGcsBucketResource sourceBucket) {
+    this.sourceBucket = sourceBucket;
+  }
+
+  @Override
+  public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
+    final ApiClonedControlledGcpGcsBucket noOpResult =
+        new ApiClonedControlledGcpGcsBucket()
+            .effectiveCloningInstructions(ApiCloningInstructionsEnum.NOTHING)
+            .bucket(null)
+            .sourceWorkspaceId(sourceBucket.getWorkspaceId())
+            .sourceResourceId(sourceBucket.getResourceId());
+    FlightUtils.setResponse(context, noOpResult, HttpStatus.OK);
```

Comment:
So this step doesn't skip anything. Maybe call it SetNullBucketCloneResponseStep or some such?
It would be nice if there was one place in the flight that populated the response.